app/controllers/searches_controller.rb

```diff
@@ -5,18 +5,51 @@ class SearchesController < CatalogController
 
   def index
     @search = FileSearch.new(params, catalog_query: self)
-    @results = @search.results
-    # page = params[:page] || 1
-    # per_page = params[:per_page] || 10
-    # resource_type = params[:t].singularize.capitalize
-    #
+    get_audios
+    get_videos
+    get_images
+    get_articles
+  end
+
+  def videos
+    get_videos
+    render layout: false
+  end
+
+  def images
+    get_images
+    render layout: false
+  end
+
+  def articles
+    get_articles
+    render layout: false
+  end
+
+  def audios
+    get_audios
+    render layout: false
   end
 
   private
 
-  # def do_search(query, filters, resource_type, page, per_page)
-  #   facets = {desc_metadata__resource_type_sim: resource_type}
-  #   facets.merge!(filters_to_query_values(filters)) unless !filters || filters.empty?
-  #   q_params = {q: query, f: facets, page: page, per_page: per_page }
-  # end
+  def get_audios
+    @audios_search = FileSearch.new(params.merge(:resource_types => ['Audio']), catalog_query: self)
```

Comment:
Line is too long. [100/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.